cloudigrade/util/azure/__init__.py

```diff
@@ -15,6 +15,10 @@
 # Reader - acdd72a7-3385-48ef-bd42-f606fba81ae7
 AZURE_READER_ROLE_ID = "acdd72a7-3385-48ef-bd42-f606fba81ae7"
 
+# Managed Services Registration assignment
+# Delete Role - 91c1777a-f3dc-4fae-b103-61d183457e46
+AZURE_MS_REG_ASSIGNMENT_DELETE_ROLE_ID = "91c1777a-f3dc-4fae-b103-61d183457e46"
```

Comment:
Should this (and the other hard-coded IDs above, now that I think about it) be loaded in from environment variables?

Will this be different for ephemeral vs. stage vs. prod? Or is this a truly universal ID across Azure? If the former, how did you create it? If the latter, where did it come from?

Reply:
this are static as they are Azure built-in role id's, they don't change.  Here's the list of them, https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles  (just a few).

Reply:
Because I'm sure to forget this, would you add a comment above these variables with that link and a succinct explanation?

Reply:
Excellent comment additions. Thank you! 🎉